### PR TITLE
[FEATURE] Added section in snippets to explain use of smarty in snippets

### PR DESCRIPTION
**Pull request by wolv-dev**

### CODE REVIEW

File: source/designers-guide/snippets/index.md [hunk 1]
```diff
@@ -59,6 +59,28 @@ While snippets are mostly used to translate plain text, they are flexible enough
...
+```
+
+Instead of changing the year of the copyright every year, make use of a smarty modifier:
+```
+Copyright {"%Y"|strftime} {config name=shopName}
+```
+
+Determine the output of your snippets with "if-queries":
+```
+{if $sArticle.length}L{$sArticle.length} {/if}{if $sArticle.length}B{$sArticle.width} {/if}{if $sArticle.length}H{$sArticle.height} {/if}cm
```
Comment (mitelg) [inline source/designers-guide/snippets/index.md]: the if's for width and height are wrong :wink:

**Comment by mitelg** [comment]
hey @wolv-dev nice contribution! :+1: 
I have one little thing :blush:

File: source/designers-guide/snippets/index.md (hunk 1 above)
Comment (wolv-dev) [reply source/designers-guide/snippets/index.md]: You're right. Just copied it from a shop.
-> Fixed!

**Comment by mitelg** [comment]
thanks @wolv-dev :+1: :tada:

**Pull request merged**